ArcGISRuntimeSDKQt_CppSamples/EditData/ContingentValues/ContingentValues.qml

```diff
@@ -0,0 +1,210 @@
+                    contingentValuesSample.updateField("Status", statusComboBox.currentValue);
+                    // Append the valid contingent coded values to the subsequent ComboBox
+                    protectionComboBox.model = [""].concat(contingentValuesSample.getContingentValues("Protection", "ProtectionFieldGroup"));
+                }
+            }
+
+            Text {
+                text: "Protection"
+                font {
+                    bold: true
+                    pointSize: 11
+                }
+            }
+
+            ComboBox {
+                id: protectionComboBox
+                model: [""];
+
+                enabled: statusComboBox.currentText !== ""
+
+                onCurrentValueChanged: {
+                    if (protectionComboBox.currentValue === "")
+                        return;
+
+                    // Update the feature's attribute map with the selection
+                    contingentValuesSample.updateField("Protection", protectionComboBox.currentValue);
+
+                    // Get the valid contingent range values for the subsequent SpinBox
+                    const minMax = contingentValuesSample.getContingentValues("BufferSize", "BufferSizeFieldGroup")
+
+                    // If getContingentValues() returned results, update the spin box values
+                    if (minMax[0] !== "") {
+                        rangeValuesSpinBox.from = minMax[0];
+                        rangeValuesSpinBox.to = minMax[1];
+
+                        // If the maxValue in the range is 0, set the buffer size to 0
+                        if (minMax[1] === 0) {
+                            contingentValuesSample.updateField("BufferSize", 0);
+                            saveButton.enabled = contingentValuesSample.validateContingentValues();
+                        }
+                    }
+                }
+            }
+
+            Text {
+                text: "Buffer Size"
+                font {
+                    bold: true
+                    pointSize: 11
+                }
+            }
+
+            Text {
+                text: rangeValuesSpinBox.from + " to " + rangeValuesSpinBox.to;
+            }
+
+            SpinBox {
+                id: rangeValuesSpinBox
+                editable: true
+                from: 0;
+                to: 0;
+                stepSize: 10
+                value: 0;
+
+                enabled: protectionComboBox.currentText !== ""
+
+                onValueChanged: {
+                    if (protectionComboBox.currentValue === "")
+                        return;
+
+                    contingentValuesSample.updateField("BufferSize", rangeValuesSpinBox.value);
+
+                    // Validate that all contingencies are valid, if so, enable the save button
+                    saveButton.enabled = contingentValuesSample.validateContingentValues();
+                }
+            }
+
+            Button {
+                id: saveButton
```

Comment:
Should the Save button be grayed out until all relevant inputs are received? Right now if there is a missing input, the save button can be pressed but it doesn't seem to do anything which may be confusing to some.